target/linux/kirkwood/config-5.4

```diff
@@ -120,7 +118,6 @@ CONFIG_DTC=y
 # CONFIG_EARLY_PRINTK is not set
 CONFIG_EDAC_ATOMIC_SCRUB=y
 CONFIG_EDAC_SUPPORT=y
-CONFIG_EXT4_FS=y
```

Comment:
EXT4 depends on JBD2, which can be removed as well

Reply:
Done. Thank You.